frontend/javascripts/admin/task/task_search_form.js

```diff
@@ -242,6 +244,15 @@ class TaskSearchForm extends React.Component<Props, State> {
             <Button style={{ marginLeft: 8 }} onClick={this.handleReset}>
               Clear
             </Button>
+            <Button
+              style={{ marginLeft: 8 }}
+              onClick={onDownloadAllTasks}
+              disabled={isLoading}
+              loading={isLoading}
+            >
+              Download settings of all tasks
```

Comment:
I'd rename this to `Download tasks as CSV`. Also, the button is in the search form now which suggests that it will download the tasks for match the current form content. However, it downloads the tasks which are currently rendered in the cell. To fix this mismatch you could either (a) move the button out of the form (and into/next to thetable somehow) or (b) make the button so that it fetches the tasks that match the criteria and download them without showing them in the table. I'd tend to (b) :)

Reply:
I implemented version b :smile: